src/Tools/MicroBuild/PublishBlobAssets.proj

```diff
@@ -18,10 +17,16 @@
   </ItemGroup>
 
   <Target Name="Build">
-    <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
-                    AccountKey="$(AccountKey)"
+    <PushToBlobFeed ExpectedFeedUrl="$(PB_PublishBlobFeedUrl)"
+                    AccountKey="$(PB_PublishBlobFeedKey)"
                     ItemsToPush="@(ItemsToPush)"
-                    Overwrite="$(PublishOverwrite)" />
+                    Overwrite="$(PublishOverwrite)"
+                    ManifestBranch="$(BUILD_SOURCEBRANCH)"
+                    ManifestBuildId="$(BUILD_BUILDNUMBER)"
+                    ManifestCommit="$(CommitHashDisplay)"
+                    ManifestName="roslyn"
+                    SkipCreateManifest="false" />
```

Comment:
`SkipCreateManifest` is optional and defaults to `false`, not needed. (But doesn't hurt anything of course.)